[EDU-6432] Edge AI reference (#1624)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/edge-ai/ai-models/florence-2.mdx

---
title: Florence-2 Large
description: >-
  Florence 2 Large is an advanced vision foundation model that leverages a prompt-based approach and a massive dataset to excel in various vision and vision-language tasks.
meta_tags: 'edge ai, ai models, artificial intelligence, edge computing'
namespace: docs_edge_ai_models_florence_2
permalink: /documentation/products/ai/edge-ai/models/florence-2/
---

> **Florence 2** is an advanced vision foundation model that leverages a prompt-based approach and a massive dataset to excel in various vision and vision-language tasks.

Reviewer: Nos outros modelos acima tu não colocou o negrito como em **Florence 2**. Padronizar?

## Model details

| Category | Details |
|----------|---------|
| **Model Name** | Florence 2 |
| **Version** | Large |
| **Model Category** | VLM |
| **Size** | 0.77B parameters |
| **HuggingFace Model** | [microsoft/Florence-2-large](https://huggingface.co/microsoft/Florence-2-large) |
| **OpenAI Compatible endpoint** | [Chat Completions](https://platform.openai.com/docs/api-reference/chat/create) |
| **License** | [MIT](https://huggingface.co/microsoft/Florence-2-large/resolve/main/LICENSE) |

## Capabilities 

| Feature | Details |
|---------|--------|
| Tool Calling | ❌ |
| Azion Long-term Support (LTS) | ❌ |
| Context Length | 4096 |
| Supports LoRA | ❌ |

> | Input data | TEXT + IMAGE |

Author: ```suggestion
| Input data | Text + Image |
```

### Tasks with no additional input:

- Whole image to natural language:

| Tag | Description |
|------|-------------|
| `<CAPTION>` | Image level brief caption |
| `<DETAILED_CAPTION>` | Image level detailed caption |
| `<MORE_DETAILED_CAPTION>` | Image level very detailed caption |

- Whole image or region to text:

| Tag | Description |
|------|-------------|
| `<OCR>` | OCR for entire image |
| `<OCR_WITH_REGION>` | OCR for entire image, with bounding boxes for individual text items |

- Whole image to regions and categories or natural language labels:

| Tag | Description |
|------|-------------|
| `<REGION_PROPOSAL>` | Proposes bounding boxes for salient objects (no labels) |
| `<OD>` | Identifies objects via bounding boxes and gives categorical labels |
| `<DENSE_REGION_CAPTION>` | Identifies objects via bounding boxes and gives natural language labels |

#### Tasks with region input
    
- Region to segment:

| Tag | Description |
|------|-------------|
| `<REGION_TO_SEGMENTATION>` | Segments salient object in a given region |

- Region to text:
    
| Tag | Description |
|------|-------------|

> | `<REGION_TO_CATEGORY>` | get object classification for bounding box |

Reviewer: ```suggestion
| `<REGION_TO_CATEGORY>` | Gets object classification for bounding box |
```

> | `<REGION_TO_DESCRIPTION>` | get natural language description for contents of bounding box |

Reviewer: ```suggestion
| `<REGION_TO_DESCRIPTION>` | Gets natural language description for contents of bounding box |
```

#### Tasks with natural language input

- Natural language to regions (one to many):

| Tag | Description |
|------|-------------|
| `<CAPTION_TO_PHRASE_GROUNDING>` | Given a caption, provides bounding boxes to visually ground phrases in the caption |

- Natural language to region (one to one):
    
| Tag | Description |
|------|-------------|

> | `<OPEN_VOCABULARY_DETECTION>` | Detect bounding box for objects and OCR text |

Reviewer: ```suggestion
| `<OPEN_VOCABULARY_DETECTION>` | Detects bounding box for objects and OCR text |
```

- Natural language to segment (one to one):

| Tag | Description |
|------|-------------|

> | `<REFERRING_EXPRESSION_SEGMENTATION>` | Referring Expression Segmentation - given a natural language descriptor identifies the segmented region that corresponds |

Reviewer: Sugestão: given a natural language descriptor identifies the segmented corresponding region, pq não ficou claro antes that corresponds...to what?

> | `<REFERRING_EXPRESSION_SEGMENTATION>` | Referring Expression Segmentation - given a natural language descriptor identifies the segmented region that corresponds |

Author: ```suggestion
| `<REFERRING_EXPRESSION_SEGMENTATION>` | Referring Expression Segmentation - given a natural language descriptor identifies the segmented corresponding region |
```
